.travis.yml

```diff
@@ -46,7 +46,13 @@ install:
     fi
   - |
     if [ "$LINT" = "1" ] || [ "$PHP" = "1" ] || [ "$DEPLOY_PLUGIN" = "1" ]; then
-      composer install
+      composer install -n --ignore-platform-reqs
```

Comment:
Why not add `--no-interaction --no-suggest` here?

Reply:
`-n` is short for `--no-interaction`. I do not see how `--no-suggest` is relevant here.

Reply:
`--no-suggest` hides all the suggestions packages make - it makes CI output more readable

Reply:
I know, but suggestions only make up for  about 4 lines in CI output that is collapsed anyway. Don't see the immediate need to change anything here.